test/projects/full/mix.exs

```diff
@@ -25,8 +25,7 @@ defmodule Full.MixProject do
       # comment to remove
       {:esbuild, "~> 0.5", runtime: Mix.env() == :dev},
       {:graphql_document, "~> 0.2.1"},
-      # test that options are retained
-      {:sourceror, "~> 0.11.1", path: "../../../deps/sourceror", override: true},
+      {:sourceror, "~> 0.14"},
```

Comment:
🤔 Uncertain how to preserve this...or if even we need to...?

Reply:
I ejected a complex app and compare the before/after and the results are identical.